src/app/component/CookiePopup/CookiePopup.style.scss

```diff
@@ -0,0 +1,45 @@
+:root {
+	--cookiepopup-bg: #{$white};
+}
+
+.CookiePopup {
+	display: flex;
+	width: 100vw;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/app/ScandiPWA/base-theme/pullRequest?prid=4649838)